hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -145,7 +146,10 @@ private void flushTransactions() {
               }
             });
 
+            long startTime = Time.monotonicNowNanos();
             omMetadataManager.getStore().commitBatchOperation(batchOperation);
+            ozoneManagerDoubleBufferMetrics.updateFlushTime(
```

Comment:
Do we need a finally block here?

Reply:
In case of exception, we miss calling updateFlushTime is to handle this scenario. I think that is fine right because anyway we got an exception, so missing updateFlushTime is fine I believe.

Reply:
> think that is fine right because anyway we got an exception, so missing updateFlushTime is fine I believe.

Fair enough. Thanks to help to understand it.